src/ape/managers/chain.py

```diff
@@ -136,12 +137,14 @@ def query(
             step=step,
             engine_to_use=engine_to_use,
         )
-
-        return self.query_manager.query(query)
+        vals = self.query_manager.query(query)
+        vals = map(lambda val: val.dict(by_alias=False), vals)
+        df = pd.DataFrame(columns=query.columns, data=vals)
+        return df
```

Comment:
Eventually should turn the lambda into a utility function
```suggestion
        vals = map(lambda val: val.dict(by_alias=False), self.query_manager.query(query))
        return pd.DataFrame(columns=query.columns, data=vals)
```